src/claim/Claim.js

```diff
@@ -187,18 +241,31 @@ class Claim extends UserCollectableAttribute {
   static getPath(identifier) {
     const { identifierComponents } = getBaseIdentifiers(identifier);
     const baseName = _.camelCase(identifierComponents[1]);
-    return `${baseName}.${identifierComponents[2]}`;
+    return baseName !== 'type' ? `${baseName}.${identifierComponents[2]}` : identifierComponents[2];
```

Comment:
why do we act specifically on the name 'type' here?

Reply:
my question also

Reply:
Types need a special handle cause they are not include in the claim path ever